picard/ui/infodialog.py

```diff
@@ -357,7 +358,12 @@ def sorttracknum(item):
             try:
                 return int(item.tracknumber)
             except:
-                return item.tracknumber or 0
+                try:
+                    # This allows to parse values like '3' but also '3/10'
+                    m = re.search('^\d+', item.tracknumber)
+                    return int(m.group(0))
+                except:
```

Comment:
We can do something better than a bare except here.

Reply:
Anything more apart from AttributeError?